RadarSDK/Radar.m

```diff
@@ -67,9 +68,15 @@ + (void)initializeWithPublishableKey:(NSString *)publishableKey options:(RadarIn
     [RadarSettings setPublishableKey:publishableKey];
 
     RadarSdkConfiguration *sdkConfiguration = [RadarSettings sdkConfiguration];
-
-    if (NSClassFromString(@"XCTestCase") == nil && options.autoLogNotificationConversions) {
-        [Radar nativeSetup];
+    // For most users not using these features, options be null and skipped,
+    //  For X-platform users initlizing Radar in the crossplatform layer, the optiosn will also be null as nativeSetup would had been called ealier 
```

Comment:
@KennyHuRadar Some typos in this comment. e.g., `initlizing` --> `initializing`, `optiosn` --> `options`, mixing `X-platform` and `crossplatform`

I realize comments and release notes don't have any functional impact on the SDK, but I think it's important that we nail the details there, otherwise we look sloppy. cc @lmeier